packages/dropdown/src/components/Dropdown.story.js

```diff
@@ -6,6 +6,22 @@ import { action } from "@storybook/addon-actions";
 import Dropdown from "./Dropdown";
 import DropdownItem from "./DropdownItem";
 
+class WithRef extends React.Component {
+  render() {
+    return (
+      <Dropdown
+        text="Select fruit"
+        handleSelection={action("handleSelection")}
+        innerRef={node => (this.triggerButton = node)}
```

Comment:
How about we use this instead?
```javascript
innerRef={action("innerRef")}
```

It's not exactly the intended use for `@storybook/addon-actions` but at least we a feedback that the ref is working.

Reply:
Nice ideia. Will do that, thanks!